### ISSUE TITLE
Do you plan to make gpcp to work with newer Java version?

### ISSUE THREAD

**Issue by ruanjiaxing** [k-john-gough/gpcp#16]
For example, Java 11? Java 8 is a LTS and will be supported for a while, so I think it's still fine in foreseeable future but sooner or later, we still have to switch to newer Java.

**Comment by btreut** [comment]
Good point. Is it possible to include/merge the [hacks](https://github.com/pahihu/gpcp-JVM)  of @pahihu? These hacks update the Java version to work with newer versions.
